planner/logical_test.go

```diff
@@ -0,0 +1,180 @@
+package planner_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/functions"
+	"github.com/influxdata/flux/planner"
+	"github.com/influxdata/flux/planner/plannertest"
+	"github.com/influxdata/flux/semantic"
+)
+
+var create = map[flux.OperationKind]planner.CreateLogicalProcedureSpec{
```

Comment:
So you're creating this map here... What would it take to use the map that gets created by all the registration that goes on in the `functions` package?  I guess that's the next step.